### ISSUE TITLE
[Bug] Insomnia loosing the data inside objects in arrays in the JSON environment declarations

### ISSUE THREAD

**Issue by IsidroFaria** [getinsomnia/insomnia#1605]
- Insomnia Version: 6.5.4
- Operating System: Windows 10

## Details
Example:
Change the Basic environment to:
```json
{
  "myArray": [
    {"a":"1"},
    {"a":"2"}
  ]
}
```
Press Done, and then open the basic environment declaration again
```json
{
  "myArray": [
    {},
    {}
  ]
}
```

**Comment by gschier** [comment]
Thanks for the report! I was able to reproduce it on my end.

Just as a note. The environment saves correctly but it's unable to display properly when the environment editor is loaded another time.

**Comment by elemanhillary** [comment]
@gschier where are environments stored,

**Comment by IsidroFaria** [comment]
Confirmed fix in version 7.0.4
